tools/caffe_converter/convert_symbol.py

```diff
@@ -128,6 +128,16 @@ def _parse_proto(prototxt_fname):
         param_string = ''
         skip_layer = False
         name = re.sub('[-/]', '_', layer.name)
+        for k in range(len(layer.bottom)):
```

Comment:
@piiswrong 
as comment on #7368 
The layer input name in some cases is changed which causes the layer added to group by mistake, for example, in vgg16, the layer fc6 input name pool5 is changed to flatten_0. Moving the state recording code to the begin of the function can fix the issue.